filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -517,36 +517,38 @@ If the specified path is not absolute, it is considered relative to the configur
 filebeat.config_dir: path/to/configs
 -------------------------------------------------------------------------------------
 
-include::../../../../libbeat/docs/generalconfig.asciidoc[]
-
-include::../../../../libbeat/docs/processors-config.asciidoc[]
-
-include::../../../../libbeat/docs/outputconfig.asciidoc[]
-
-include::../../../../libbeat/docs/shared-path-config.asciidoc[]
-
-include::../../../../libbeat/docs/loggingconfig.asciidoc[]
-
-
 ===== shutdown_timeout
 
-`shutdown_timeout` defines how long filebeat waits on shutdown for the publisher to finish.
+How long Filebeat waits on shutdown for the publisher to finish.
 
-In case the defined output is blocked and does not confirm all events, filebeat keeps resending the events
-before the confirmation is persisted to disk in the registrar. Waiting on the output to acknowledge all events
-can take seconds or much longer depending on the state of the output. To still be able to shutdown filebeat, filebeat
-does not wait for the confirmation of all events on shutdown. This means the state for events which were sent but
-not acked yet, will not be persisted to disk. This can lead to the issue that some events are sent again on starting
-filebeat afterwards as the events ended up at the receiving end but filebeat was already stopped when the acked arrived.
+When the defined output is blocked and has not confirmed all events, Filebeat keeps resending the events
+until the confirmation is persisted to disk in the registrar. Depending on the state of the output,
+waiting on the output to acknowledge all events can take seconds or much longer. To ensure that Filebeat is able
```

Comment:
The whole sentence I wrote is a little bit messy. Perhaps it makes sense to "rewrite" the sentence and shorten it instead of trying to fix it? @dedemorton Suggestion?